cumulusci/tasks/robotframework/robotdoc.py

```diff
@@ -0,0 +1,98 @@
+import os
+import os.path
+import time
+import jinja2
+import robot.utils
+
+import cumulusci
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.core.tasks import BaseTask
+from cumulusci.core.utils import process_list_arg
+
+from robot.libdocpkg import LibraryDocumentation
+from robot.libraries.BuiltIn import RobotNotRunningError
+
+
+class RobotDoc(BaseTask):
+    task_options = {
+        "outputdir": {
+            "description": "The path to a folder where generated files will be placed",
+            "required": True,
+        },
+        "files": {
+            "description": "A list of keyword definition files (eg: .py, .robot, .resource)",
+            "required": True,
+        },
```

Comment:
I think we talked about giving the task the path to a folder and making it able to discover the keyword libraries contained in the folder. Did you decide that won't work for some reason? It would require less thought when running the task, and we could provide `tests/` as a default path that would generally work for our products without requiring option overrides in the cumulusci.yml for each product.

Reply:
Did I decide it didn't work? No, not really. I just chose the simpler route for the time being. I've got code that can detect, but wan't sure if we wanted to stick with "explicit is better than implicit" in this case. I'm open to either. I definitely see value in it working right out of the box without the user having to edit their config file.

It will be pretty easy for me to switch the code to auto-detect.

Reply:
I haven't added the auto-detect code yet, and I think I'd prefer that we save that for a new release. This little project has already seen a some feature creep. I say we leave auto-detect out for the moment, and perhaps create a separate story for some enhancements in the interest of getting this out the door.